NOTICE

```diff
@@ -0,0 +1,28 @@
+j2objc-common-libs-e2e-test - https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test
+  a subproject of j2objc-gradle - Gradle plugin for j2objc (Java to Objective-C transpiler)
+  https://github.com/j2objc-contrib/j2objc-gradle
+
+Copyright (c) 2015 the authors of j2objc-gradle
+
+Main Contributors:
+  Advay Mengle @advayDev1 <source@madvay.com>
+  Bruno Bowden @brunobowden <github@brunobowden.com>
+  Michael Gorski @confile <mail@michaelgorski.de>
+  
+Thanks:
+  Peter Niederweiser @pniederw <peter@pniederw.com>
+  Sterling Greene @big-guy <sterling.greene@gradleware.com>
```

Comment:
Shall we drop the people that haven't contributed to this repo? Makes sense to leave my name as you'll undoubtedly get contributions from me.